sdk/sdk-api/src/main/java/mil/emp3/api/interfaces/core/IStorageManager.java

```diff
@@ -63,6 +63,8 @@
 
     void addOverlays(IMap map, List<IOverlay> overlays, boolean visible, Object userContext) throws EMP_Exception;
 
+    void apply(IMapInstance mapInstance, IFeature feature, Object userContext) throws EMP_Exception;
```

Comment:
I noticed there was no javadoc in this entire class. We should add ticket to add the documentation for the rest of the methods in this class, but could you add the documentation for the method you have written please.

Reply:
I added it, but the better solution would be to get rid of this interface.  It is internal and there is no reason for it.